README-slurm.md

```diff
@@ -69,16 +69,16 @@ Slurm-specific Configuration:
 
 Job-sizing:
 
-- `ligands_todo_per_queue`: This determines how many ligands should be processed at a minimum per job. A value of '10000' would mean that each subjob with `slurm_cpus` number of CPUs should dock this number of ligands prior to completing. In general jobs should run for approximately 30 minutes or more. How long each docking takes depends on the receptor, ligand being docked, and docking program-specific settings (such as `exhaustiveness`). Submitting a small job to determine how long a docking will take is often a good idea to size these before large runs.
+- `dockings_todo_per_queue`: This determines how many ligands should be processed at a minimum per job. A value of '10000' would mean that each subjob with `slurm_cpus` number of CPUs should dock this number of ligands prior to completing. In general jobs should run for approximately 30 minutes or more. How long each docking takes depends on the receptor, ligand being docked, and docking program-specific settings (such as `exhaustiveness`). Submitting a small job to determine how long a docking will take is often a good idea to size these before large runs.
 
 
 The ligands to be processed should be included in the file within `tools/templates/todo.all`. This file can be automatically generated from the VirtualFlow website.
 
 
 ### Data for Virtual Screening
 
-The location of the collection files to be used in the screening should be located in `collection_folder` (defined in `all.ctrl`).
-
+The location of the collection files (containing the list of processed ligands) to be used in the screening should be located in `collection_folder` (defined in `all.ctrl`). Typically, this is set to the input folder (`../input-files/ligand-library`).
+Apart from the input ligands, all other inputs used for docking (such as the receptor file and docking config file) are located as subdirectories within the `collection_folder`.
```

Comment:
Those files are in `docking_scenario_basefolder`

Reply:
AHH!